scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
```

Comment:
:stop_sign: **Security Vulnerability**: Command injection vulnerability detected. This allows potential attackers to execute arbitrary commands on the system[^1].

[^1]: CWE-78: Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection') - https://cwe.mitre.org/data/definitions/78.html
```suggestion
// Use a safer method to execute commands
import { execFile } from 'child_process'
import { promisify } from 'util'

const execFileAsync = promisify(execFile)

async function executeCommand(userInput: string) {
    try {
        const { stdout } = await execFileAsync('ls', [userInput])
        console.log(stdout)
    } catch (error) {
        console.error('Error executing command:', error)
    }
}
```